src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.box {
+  width: 350px;
+  background-color: #308dc0;
+  border: 1px solid #e75551;
+  margin: 20px;
+  margin-bottom: 40px;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+.orange_box {
+  background-color: #ffa500;
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.grey_box {
```

Comment:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state. In the case with the name _red_, _green_, maybe in the future, the color of your block will be changed, so you have to rename all classes on the page.

Reply:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.